### DISCUSSION TITLE
Feature suggestion (add 30min option to 30 day display)

### DISCUSSION THREAD

**Discussion by cy754** [opennem/opennem#218] [Ideas] [answered]
I've been trying to find a way of suggesting a feature for the OpenNEM web site, hopefully this is the right place (apologies if not).

It would be great if there was a "30min" option for the "30day"displays. This would equate to 1440 data points, which should be doable considering the 7day/5min combination represents 2016 data points. 

Thanks.

**Comment by simonhac** [comment, marked as answer]
thanks @cy754, you found the right place for a feature request. thank you!

i think it's a good idea. we'll discuss internally. opened as #219.

here's what germany looks like over ~30 days @ ~30 min resolution:
<img width="754" alt="image" src="https://user-images.githubusercontent.com/20500649/223875870-66d0144d-5d1c-4149-a7fb-d90ce35ccae9.png">

from [energy-charts](https://www.energy-charts.info/charts/power/chart.htm?l=en&c=DE&interval=month&month=02)

**Reply by cy754** [reply]
Excellent. Looks Perfect. As an old programmer (since punched cards), I admit it was starting to drive me a little bit crazy (I use the graphs to show to others how well renewables are going, especially where I am in Adelaide). I can put my mind at ease now. Keep up the great work.
